frontend/src/components/Header.jsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import { FaBars, FaTimes } from "react-icons/fa";
+import headerArrow from "../images/global-header-arrow.svg";
+import headerBook from "../images/global-header-book.svg";
+import headerDoor from "../images/global-header-door.svg";
+import headerLogo from "../images/global-header-logo.svg";
+import headerMagnifyglass from "../images/global-header-Magnifyglass.svg";
+import headerNew from "../images/global-header-new.svg";
+import headerSmile from "../images/global-header-smile.svg";
+import headerStar from "../images/global-header-star.svg";
+
+export const Header = ({ children }) => {
+  return (
+    <header className="flex flex-row w-full  pl-5 pr-5  max-w-[1440px] mx-auto  justify-between items-center  text-lg font-arialRegular   content-center h-20 bg-black text-stone-50  xs:pl-2 xs:pr-2  md: text-md md:pr-2 md:pl-2  ">
+      {children}
+    </header>
+  );
+};
+
+Header.Logo = () => {
+  return (
+    <img
+      className="w-80 pt-4 xs:w-60 md:w-70 "
+      src={headerLogo}
+      alt="slopopedia logo"
+    />
+  );
+};
+
+Header.NavLinks = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleMenu = () => {
+    setOpen((prev) => !prev);
+  };
+
+  const navLinks = [
```

Comment:
I think it would be ideal to pass this array into this component as a prop, but I'm going to go ahead and approve/merge this for now to keep things rolling since this is a minor refactor and the only thing I'd change about this PR